src/groups/mwc/mwcc/mwcc_orderedhashmap.t.cpp

```diff
@@ -286,8 +286,8 @@ static void test4_rinsert()
         RcType rc = map.rinsert(bsl::make_pair(i, i + 1));
         ASSERT_EQ_D(i, true, rc.second);
         ASSERT_EQ_D(i, true, rc.first != map.end());
-        ASSERT_EQ_D(i, i, rc.first->first);
-        ASSERT_EQ_D(i, (i + 1), rc.first->second);
+        ASSERT_EQ_D(i, static_cast<int>(i), rc.first->first);
+        ASSERT_EQ_D(i, static_cast<int>(i + 1), rc.first->second);
```

Comment:
I think that declaring `i` as `int` might be more elegant since we correctly set the type corresponding to the map iterator value from the very beginning.

On comparison, with cast here, it seems that we set up the type incorrectly in the beginning and then try to patch it on every line.

Reply:
The same could be applied for other places in this file